components/domains/termCheckbox.tsx

```diff
@@ -21,6 +22,7 @@ const TermCheckbox: FunctionComponent<TermCheckboxProps> = ({
           variant === "white" ? styles.whiteCheckbox : styles.defaultCheckbox
         }
         onClick={onChange}
+        checkedIcon={ < CustomCheckmarkIcon /> }
```

Comment:
Please remove unwanted blank spaces (like that: checkedIcon={<CustomCheckmarkIcon />})